src/components/dataset/schemaCreation/DatasetSchemaInformationView.tsx

```diff
@@ -0,0 +1,334 @@
+import React, { useMemo, useState } from 'react';
+import ReactDOMServer from 'react-dom/server';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import { WithStyles, withStyles } from '@mui/styles';
+import {
+  Typography,
+  TextField,
+  CustomTheme,
+  Grid,
+  Select,
+  MenuItem,
+  Autocomplete,
+} from '@mui/material';
+import { clsx } from 'clsx';
+import { TdrState } from 'reducers';
+import { Controller, useFormContext } from 'react-hook-form';
+import SimpleMDE from 'easymde';
+import { SimpleMdeReact } from 'react-simplemde-editor';
+import { GCP_REGIONS, AZURE_REGIONS } from 'constants/index';
+import isEmail from 'validator/lib/isEmail';
+import WithoutStylesMarkdownContent from '../../common/WithoutStylesMarkdownContent';
```

Comment:
Optional Nit: It looks like generally y'all use absolute paths rather than relative? Should this be an absolute path?

Reply:
(@snf2ye What do you think?)

I haven't paid much attention to the imports, but I assumed this was okay?